torch/_dynamo/symbolic_convert.py

```diff
@@ -450,8 +450,22 @@ def wrapper(self: "InstructionTranslatorBase", inst: Instruction):
                 excp.remove_from_stats()
                 excp.add_to_stats("graph_break")
                 reason = GraphCompileReason(excp.msg, user_stack)
+
             self.restore_graphstate(state)
 
+            # This code is not *just* and optimization! Code below, specifically,
+            # the tensor-stack-only-fastpath in compile_subgraph make assumptions
+            # about the state of the graph. It assumes that all the tensors in the stack are
+            # valid to write to output. This is generally sound, UNLESS (1) there is nothing else
+            # in the graph and (2) you failed to create placeholders (the caue of the graph break)
+            # This is a rare edge state, and so we apply the same logic here as we do
+            # when we hit a RETURN_VALUE for a frame we deem useless.
```

Comment:
Self note: I do wonder if there's a tighter fix above, where we handle arg creation failures speshul and handle this from there, instead of under break_graph_if_unsupported. Starting a graph break and turning it into a skip feels dirty, and it is, we arguably missed something earlier and should give it its own path (not unlike failures during InstructionTranslator creation having their own distinct path), Would be good to document and cleanup lifecycles / state checkpoints in the entire system.